bendigobank.rb

```diff
@@ -0,0 +1,113 @@
+require 'watir'
+require 'json'
+require 'nokogiri'
+require 'date'
+require_relative 'account.rb'
+require_relative 'transaction.rb'
+
+class Bendigobank
+  URL = "http://demo.bendigobank.com.au"
+
+  def self.execute
+    connect
+    fetch_accounts
+    fetch_transactions
+    show_output
+  end
+
+  def self.connect
+    Watir.logger.ignore :deprecations
+    @browser = Watir::Browser.new :firefox
+    @browser.goto URL
+    @browser.button(name: 'customer_type').click
+  end
+
+  def self.fetch_accounts
+    html = Nokogiri::HTML.fragment(@browser.ol(class: "grouped-list grouped-list--compact").html)
+    parse_accounts(html)
+  end
+
+  def self.fetch_transactions
+    to_date = Time.new().to_datetime
+    from_date = to_date << 2
+
+    to_date = format_date(to_date)
+    from_date = format_date(from_date)
+
+    @browser.ol(class: 'grouped-list__group__items').each_with_index do |li,i|
+      li.a(class: 'panel--hover').click
+      @browser.i(class: 'ico-nav-bar-filter_16px').click
+      @browser.a(class: 'panel--bordered__item').click
+      @browser.ul(class: 'radio-group').li(index: 8).click
+      @browser.text_field(name: 'toDate').set(to_date)
+      @browser.text_field(name: 'fromDate').set(from_date)
+      @browser.button(class: 'button--primary').click
+      @browser.button(class: 'button--primary').click
+      until @browser.div(class: '_3Wd5wOSiEN').present?
```

Comment:
Try to no use hardcoded values.  

`until @browser.p(text: "No more activity").present?`